### PR TITLE
Add links to definition of substantive change

### LINKED ISSUE #233: Define "substantive"
The IPR failure page has a button to ["Mark as non-substantive"](https://w3c.github.io/repo-management.html#:~:text=mark,non%2Dsubstantive), but there's no information about what that means. I'm also having trouble finding any other documentation that mentions "substantive", for example in https://www.w3.org/community/about/faq/ or https://www.w3.org/community/about/process/cla/.

Since it's generally non-lawyers clicking this button, it'd be nice to include some guidance about when doing so is ok.

### PR DESCRIPTION
**Pull request by dontcallmedom**
close #233

### CODE REVIEW

File: application/pr/viewer.jsx [hunk 1]
```diff
@@ -120,7 +120,7 @@ export default class PRViewer extends React.Component {
             if (st.pr.acceptable === "yes") {
                 let revert = "";
                 if (st.pr.markedAsNonSubstantiveBy) {
-                     revert = <span><button name="substantive" onClick={this.markSubstantiveOrNot.bind(this)}>Unmark as non-substantive</button></span>;
+                     revert = <span><button name="substantive" onClick={this.markSubstantiveOrNot.bind(this)}>Unmark as non-substantive</button><a href="https://www.w3.org/Consortium/Process/#substantive-change" title="definition of a substantive change in the W3C Process">🛈</a></span>;
```
Comment (deniak) [inline application/pr/viewer.jsx:123]: It seems that unicode character requires to install some other fonts. Maybe switching to ℹ could be more convenient.

**Review by deniak — APPROVED** [review]
I left a small comment on the unicode character but otherwise, LGTM.

File: application/pr/viewer.jsx (hunk 1 above)
Comment (dontcallmedom) [inline application/pr/viewer.jsx:123]:
```suggestion
                     revert = <span><button name="substantive" onClick={this.markSubstantiveOrNot.bind(this)}>Unmark as non-substantive</button><a href="https://www.w3.org/Consortium/Process/#substantive-change" title="definition of a substantive change in the W3C Process">ℹ </a></span>;
```

File: application/pr/viewer.jsx [hunk 2]
```diff
@@ -137,7 +137,7 @@ export default class PRViewer extends React.Component {
                     nplcUrl.search = qs;
                     nplc = <a className="button" target="_blank" href={nplcUrl}>Ask for non-participant commitment</a>
                 }
-                action = <span><button  onClick={this.revalidate.bind(this)}>Revalidate</button> <button name="nonsubstantive" onClick={this.markSubstantiveOrNot.bind(this)}>Mark as non-substantive</button>{nplc}</span>;
+                action = <span><button  onClick={this.revalidate.bind(this)}>Revalidate</button> <button name="nonsubstantive" onClick={this.markSubstantiveOrNot.bind(this)}>Mark as non-substantive</button><a href="https://www.w3.org/Consortium/Process/#substantive-change" title="definition of a substantive change in the W3C Process">🛈</a>{nplc}</span>;
```
Comment (dontcallmedom) [inline application/pr/viewer.jsx:140]:
```suggestion
                action = <span><button  onClick={this.revalidate.bind(this)}>Revalidate</button> <button name="nonsubstantive" onClick={this.markSubstantiveOrNot.bind(this)}>Mark as non-substantive</button><a href="https://www.w3.org/Consortium/Process/#substantive-change" title="definition of a substantive change in the W3C Process">ℹ </a>{nplc}</span>;
```

**Comment by deniak** [comment]
I just [rebuilt app.js](https://github.com/w3c/ash-nazg/commit/8706db33965a381fa4b391cd59f92d0658faa6ae). Now deployed.

**Pull request merged**